packages/react/src/components/ImageGalleryModal/ImageTile.jsx

```diff
@@ -78,6 +83,7 @@ const ImageTile = ({
               event.stopPropagation();
               onDelete(id);
             }}
+            testId={`${testId}-delete-button`}
```

Comment:
this one is not overriding a default, because previously this component was using the CarbonButton which does not have a built-in default testId.